tests/macsec/test_deployment.py

```diff
@@ -23,3 +26,45 @@ def test_config_reload(self, duthost, ctrl_links, policy, cipher_suite, send_sci
         assert wait_until(300, 6, 12, check_appl_db, duthost, ctrl_links, policy, cipher_suite, send_sci)
         # Recover the original config file
         duthost.shell("sudo cp config_db.json /etc/sonic/config_db.json")
+
+    @pytest.mark.disable_loganalyzer
+    def test_scale_rekey(self, duthost, ctrl_links, rekey_period, wait_mka_establish):
+        dut_egress_sa_table_orig = {}
+        dut_ingress_sa_table_orig = {}
+        dut_egress_sa_table_current = {}
+        dut_ingress_sa_table_current = {}
+        new_dut_egress_sa_table = {}
+        new_dut_ingress_sa_table = {}
+
+        # Shut the interface and wait for all macsec sessions to be down
+        for dut_port, nbr in ctrl_links.items():
+            _, _, _, dut_egress_sa_table_orig[dut_port], dut_ingress_sa_table_orig[dut_port] = get_appl_db(
+                duthost, dut_port, nbr["host"], nbr["port"])
+            intf_asic = duthost.get_port_asic_instance(dut_port)
+            duthost.shell("config interface {} shutdown {}".format(intf_asic.cli_ns_option, dut_port))
```

Comment:
use the function in sonic_asic.py iso of CLI commands here https://github.com/sonic-net/sonic-mgmt/blob/aa1d42a81941fda710310a4b46a33c6c71684741/tests/common/devices/sonic_asic.py#L535